src/components/BaseHeading.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const BaseHeading = ({ level, text }) => {
+  const HeadingTag = `h${level}`
+
+  return (
+    <HeadingTag className="text-4xl text-center md:text-6xl">
```

Comment:
issue(nb, this won't impact us just yet 😄 ): Let's make the style dependent on the heading level. We can see in the Figma that `text4xl` and `md:text-6xl` apply to `h1`, but what about when we want to use this component for `h2`, `h3`, etc? Let's make those styles dynamic.